sds/src/scanner/test/match_validation.rs

```diff
@@ -268,6 +268,30 @@ fn test_mock_http_timeout() {
         _ => assert!(false),
     }
 }
+
+#[test]
+fn test_matches_from_rule_without_validation_are_not_ignored() {
```

Comment:
minor: The name could be more explicit
```suggestion
fn test_matches_from_rule_without_validation_have_not_available_match_validation_status() {
```